.gitignore

```diff
@@ -0,0 +1,3 @@
+_site/
+.sass-cache/
+./Gemfile.lock
```

Comment:
Remove the ./ So that you won't commit the Gemfile.lock

Reply:
OK Sir .

Reply:
Done @abishekvashok  . Please review .